arcgis-ios-sdk-samples/Layers/Apply unique values with alternate symbols/ApplyUniqueValuesAlternateSymbolsViewController.swift

```diff
@@ -0,0 +1,118 @@
+// Copyright © 2022 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class ApplyUniqueValuesAlternateSymbolsViewController: UIViewController {
+    @IBOutlet var mapView: AGSMapView! {
+        didSet {
+            // Set the map and its initial viewpoint.
+            mapView.map = AGSMap(basemapStyle: .arcGISTopographic)
+            mapView.setViewpoint(AGSViewpoint(center: AGSPoint(x: -13632095.660131, y: 4545009.846004, spatialReference: .webMercator()), scale: 7500))
```

Comment:
```suggestion
            mapView.setViewpoint(AGSViewpoint(center: AGSPoint(x: -13632095.660131, y: 4545009.846004, spatialReference: .webMercator()), scale: 7_500))
```
Can also be

```swift
mapView.setViewpointCenter(AGSPoint(x: -13632095.660131, y: 4545009.846004, spatialReference: .webMercator()), scale: 7_500)
```

Reply:
As this viewpoint is used in multiple places, you can store it as a instance property.